x-pack/plugins/security_solution/common/api/timeline/resolve_timeline/resolve_timeline_route.schema.yaml

```diff
@@ -15,7 +15,8 @@ paths:
       x-labels: [serverless, ess]
       x-codegen-enabled: true
       operationId: ResolveTimeline
-      summary: Get an existing saved timeline or timeline template.
+      summary: Resolve a Timeline or Timeline template
+      description: 
```

Comment:
There’s already a 'Get an existing saved timeline or timeline template' operation elsewhere (GET /api/timeline), so the existing summary doesn't seem right. 🤔 Could someone confirm if this suggestion (Resolve a Timeline or Timeline template) would be correct here and help provide an operation description (what does 'resolving' a Timeline mean)?

Reply:
That is a great catch. I do not also see much difference between the two 😕 🤔 . I will check it with the team, may we can remove one of them. But I guess it should block this PR.

Maybe @michaelolo24 and @kqualters-elastic   are aware of the difference.

Reply:
Thanks, @logeekal! We also don't have this operation (/api/timeline/resolve) documented in our [ESS API docs](https://www.elastic.co/guide/en/security/current/timeline-api-overview.html), so it wasn't clear to me what it actually does. 
If you think investigation / making a decision on this might take a while, I'm OK with leaving it as it is for now to unblock this PR while the team looks into it.

Reply:
Sorry I originally said `But I guess it should block this PR` while I meant to say `But I guess it should *not* block this PR`. I think you can go ahead and merge this PR and we will discuss and remove it or add docs for it if there is a difference.

Reply:
Sounds good, ty!